README.md

```diff
@@ -215,3 +215,14 @@ browserify({debug: true}).transform("babelify", {sourceMaps: false});
 ```sh
 $ browserify -d -t [ babelify --no-sourceMaps ]
 ```
+
+### Can I use private/different babel package (like scoped babel 7)?
+
+Yes, just pass it along as `babel` option, you still need to install `babel-core <= 6` as babelify uses support functions that were removed on babel 7.
+
+```js
+browserify().transform("babelify", {
+  presets: ["@babel/preset-env"],
+  babel: require("@babel/core")
```

Comment:
I don't think this is a good idea. If we're going to support 7, we should just add it directly. The utilities that were removed can easily be implemented inside this module on their own, if we actually want to support them.

Reply:
I didn't want to break the current implementation as there were breaking changes in babel 7 (everything became scope), maybe we can add it to the next major version.